sdk/python/tests/conftest.py

```diff
@@ -94,8 +93,6 @@ def pytest_collection_modifyitems(config, items: List[Item]):
     should_run_integration = config.getoption("--integration") is True
     should_run_benchmark = config.getoption("--benchmark") is True
     should_run_universal = config.getoption("--universal") is True
-    should_run_goserver = config.getoption("--goserver") is True
```

Comment:
why are we removing the `goserver` mark? I feel like it will still be useful from a developer perspective to be able to run the specific tests that rely on the go server

if you feel strongly that you want to remove `goserver`, you should also remove `goserver` from Makefile

Reply:
Yeah, the idea was not to have go dedicated tests at all, and instead have dedicate test environment with flag. Currently the only blocker is support of ODFV on Go side. So I will rollback this for now, until tests are merged.